# worktree hvoya-trial-rangerrick at d7cb806e1 (linux-trial/rangerrick); uncommitted diff saved before removal 2026-10-16
diff --git a/Examples/IPlugEffect/CMakeLists.txt b/Examples/IPlugEffect/CMakeLists.txt
index 7163d74c4..25e642df8 100644
--- a/Examples/IPlugEffect/CMakeLists.txt
+++ b/Examples/IPlugEffect/CMakeLists.txt
@@ -22,7 +22,8 @@ find_package(iPlug2 REQUIRED)
 iplug_add_plugin(${PROJECT_NAME}
   SOURCES
     IPlugEffect.cpp
-    IPlugEffect.h
+    IPlugEffectUI.cpp
+    IPlugEffect.hpp
     resources/resource.h
   RESOURCES
     resources/fonts/Roboto-Regular.ttf
